docs/UserGuide.md

```diff
@@ -147,6 +147,42 @@ Clears all entries from the address book.
 
 Format: `clear`
 
+### Adding new Project : `addP`
+
+Adds a new project in TaskHub
+
+Format: `addP p/PROJECT_NAME [e/EMPLOYEE_NAME] ...`
+
+* Adds a new project with the employees assigned to the project.
+* The employee must exist in the contacts list. 
+
+Examples: 
+* `addP p/Project1 e/Betsy` will add `Project1` to the projects list with `Betsy` assigned to the project.
+* `addP p/Project2` will add an empty `Project2` to the projects list.
+
+### Assign Employee to Project: `assignE`
```

Comment:
Following the standard of other commands in the UG, I think it is better to change this:

- `Assign Employee to Project:` -> `Assign employee(s) to a project`

There are other instances where this occurs, and it is best to change them (namely in line 150 and 174).